frontend/src/tests/components/eMIB/EditActionDialog.test.js

```diff
@@ -210,3 +210,29 @@ function testMode(actionType, editMode) {
     );
   }
 }
+
+it("check that email duplication bug is no longer an issue", () => {
```

Comment:
Can this name be more general so that the context makes sense no matter what?
"clicking on the button only adds the email once"

Reply:
Changed in 130622c